schemas/ns10_cache_entry.fbs

```diff
@@ -0,0 +1,13 @@
+
+file_identifier "ns10";
+
+
+/// pylint: skip-file
+table CacheEntryFB {
+        time:double;
+	ttl:double;
```

Comment:
What does `ttl` mean here? Could you use a more explicit name?

Reply:
`ttl` stands for 'Time to live'. This `CacheEntryFB` corresponds exactly to class called `CacheEntry` in NICOS. I do not want to change the variable names as it will confuse people reading the code in NICOS. But, I can certainly add comments here.